packages/flutter_driver/lib/src/timeline_summary.dart

```diff
@@ -173,8 +163,26 @@ class TimelineSummary {
         .reduce((double a, double b) => math.max(a, b));
   }
 
-  List<TimedEvent> _extractFrameEvents() => _extractCompleteEvents('Frame');
+  double _averageInMillis(List<Duration> durations) {
```

Comment:
This is almost the same as `_averageDurationMillis`. Instead of duplicating the functionality, could we turn the "tdur" event into `TimedEvent` and reuse `_averageDurationInMillis`? These events must have the `ts` field, so you can `new TimedEvent(ts, ts + tdur)` no? Or perhaps add a new constructor `new TimedEvent.fromDuration(ts, tdur)`?